exercises/sum-of-multiples/src/example.c

```diff
@@ -0,0 +1,24 @@
+#include "sum_of_multiples.h"
+#include <stdbool.h>
+
+unsigned int sum_of_multiples_to(const unsigned int *multiples, const size_t number_of_multiples, const int upper_bound)
```

Comment:
`upper_bound` and `*multiples` should have the same type. Same goes for `current` in the loop below.